app/views/content/steps-to-become-a-teacher/_step_2_check_your_qualifications.html.erb

```diff
@@ -1,4 +1,4 @@
-<p>Once you have a degree or equivalent qualification you’re ready for postgraduate primary or secondary initial teacher training courses.</p>
+<p>Once you have a degree or equivalent qualification you’re ready for postgraduate primary or secondary initial teacher training courses. You will get the teaching qualifications you need by completing your course.</p>
 
 <p>You also need the following GCSEs at grade 4 (C) or above, or equivalent qualifications:</p>
```

Comment:
```suggestion
<p>You also need the following GCSEs at grade 4 (C) or above:</p>
```
Rather than saying 'or equivalent qualification' every time we mention a qualification, would it not be simpler to have one sentence explaining this further down (see comment on 'you may be able to show you meet the standard...')